qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -419,6 +420,8 @@ def jobs(
                       specified in ``job_tags`` to be included.
                     * If "OR" is specified, then a job only needs to have any
                       of the tags specified in ``job_tags`` to be included.
+            descending: If ``True``, return the jobs in descending order of the job
+                creation date (newest first). If ``False``, return in ascending order.
```

Comment:
This docstring *slightly* differs from `IBMBackendService.jobs`, it's probably best to synchronize them